README.md

```diff
@@ -64,7 +89,10 @@ argument. Finally, demonstrate calling `doSomething` with a function.**
 
 Your Answer:
 ```js
-// write code here
+// function doSomething(thingToDo) {
+// return "Do" + " " + "thingToDo";
```

Comment:
This function is always going to return the string "Do thingToDo". You want it to actually **do** `thingToDo` -- not just print out the words "thingToDo". Please read through this: http://cwbuecheler.com/web/tutorials/2013/javascript-callbacks/